test_results.py

```diff
@@ -0,0 +1,172 @@
+import unittest
+import sys
+import os
+
+# Add the parent directory to the Python path
+sys.path.append(os.path.dirname(os.path.dirname(os.path.abspath(__file__))))
+
+from utils.custom_tsv_parser import ParseTSVFile
+
+
+def get_scode_cscode_id(data):
+    # Create an id with scode and center with sorted values of scode and center
+    school_centers = []
+    for row in data:
+        scode = int(row["scode"])
+        center = int(row["cscode"])
+        sccode_center_id = sorted((scode, center))
+        sccode_center_id = "_".join(map(str, sccode_center_id))
+        school_centers.append(sccode_center_id)
+    return school_centers
+
+
+class TestSchoolCenter(unittest.TestCase):
+    """_Tests to validate the outcome of the output are matching
+    as per the requirements
+
+    Needs the ouptut from the results/school-center.tsv and
+    results/school-center-distance.tsv files to be present
+
+    """
+
+    def setUp(self):
+        self.school_center_file = "results/school-center.tsv"
```

Comment:
do we need to run the application manually so that this file is generated ? can running the script done as part of test setup ?

Reply:
Good Idea I will add the results generation as par of the setup itself

Reply:
Done